mn/user-app.js

```diff
@@ -0,0 +1,77 @@
+/****************************************************************************************
+ * MONGOLIAN
+ ****************************************************************************************/
+export default {
+    'socialLoginTitle': 'Now4real Сошиал нэвтрэлт',
+    'emailLoginTitle': 'Now4real Имэйл нэвтрэлт',
+    'editProfileTitle': 'Профайл засварлах - Now4real',
+
+    'genericError': 'Уучлаарай алдаа гарлаа',
+    'sessionExpiredError': 'Холболт салсан тул дахин хандна уу',
+    'recaptchaError': 'Уучлаарай, таны хандалт сэжигтэй байна',
+    'sendThrottlingError': 'Шинэ код авахын тулд өмнөх имэйл илгээснээс хойш хамгийн багадаа $N минутын дараа дахин хандна уу',
+
+    /**
+     * Email signin step 1
+     */
+    'emailSigninStep1.title': 'Имэйл хаягаар нэвтрэх',
+
+    'emailSigninStep1.emailInfo': 'Имэйл хаягаа оруулна уу',
+    'emailSigninStep1.emailPlaceholder': 'Имэйл',
+
+    'emailSigninStep1.continue': 'Үргэлжлүүлэх',
+
+    'emailSigninStep1.invalidEmailError': 'Зөв имэйл хаяг оруулна уу',
+
+    /**
+     * Email signin step 2
+     */
+    'emailSigninStep2.title': 'Имэйл баталгаажуулалт',
+
+    'emailSigninStep2.info1': '($E) имэйл хаягаар баталгаажуулалтын код илгээсэн болно',
+    // $E is the user email
+    // example: "We sent you a verification code to your email address (john@mail.test)"
+
+    'emailSigninStep2.info2': 'Код хүлээж аваагүй юу?',
+
+    'emailSigninStep2.resend': 'Дахин илгээх',
+    'emailSigninStep2.confirm': 'Батлах',
+
+    'emailSigninStep2.otpResendFeedback': 'Шинэ код илгээсэн',
+    'emailSigninStep2.invalidOtpError': 'Хүчингүй эсвэл хугацаа дууссан код байна',
+
+    /**
+     * Email signin step 3
+     */
+    'emailSigninStep3.title': 'Профайлын мэдээллээ гүйцэт оруулна уу',
+
+    'emailSigninStep3.feedback': 'Код баталгаажсан. Сүүлийн алхам...',
+
+    'emailSigninStep3.nameInfo': 'Дэлгэцэнд харагдах нэр (нийтийн чатанд харагдах боно)',
+    'emailSigninStep3.namePlaceholder': 'Нэр',
+
+    'emailSigninStep3.emailInfo': 'Таны имэйл хаяг (бусдад харагдахгүй)',
+
+    'emailSigninStep3.continue': 'Үргэжлүүлэх',
+
+    'emailSigninStep3.invalidNameError': 'Хүчингүй нэр',
+
+    /**
+     * Edit profile
+     */
+    'editProfile.title': 'Профайл засварлах',
+
+    'editProfile.pictureInfo': 'Таны зураг (нийтийн чатанд харагдна)',
+
+    'editProfile.nameInfo': 'Дэлгэцийн нэр (нийтийн чатанд харагдна)',
+    'editProfile.namePlaceholder': 'Нэр',
+
+    'editProfile.emailInfo': 'Таны имэйл (Бусдад харагдахгүй)',
+
+    'editProfile.updatedFeedback': 'Таны мэдээллийг шинэчилж дууслаа. Энэ цонхыг хаана уу',
+
+    'editProfile.gravatarInfo': 'Gravatar ыг ашиглан таны зургийг харуулдаг өөрчлөхийн тулд <a>Gravatar/WordPress -т бүртгүүлнэ үү</a> .',
```

Comment:
Could you underline that Gravatar is an external service?

Reply:
Updated.